src/terrain/terrain.hpp

```diff
@@ -337,6 +337,11 @@ class Terrain : public TerrainBase {
         return chunks_;
     }
 
+    [[nodiscard]] inline const Chunk&
+    get_chunk(size_t i) const {
+        return chunks_[i];
```

Comment:
```suggestion
	assert(i < chunks_.size());
        return chunks_[i];
```
don't worry, asserts are disabled on `Release` builds